lxd/images.go

```diff
@@ -547,20 +546,19 @@ func imageExport(d *Daemon, r *http.Request) Response {
 	path := shared.VarPath("images", hash)
 	filename := imgInfo.Filename
 
-	// test compression, for content type header
-	// if unknown compression we send standard header
-	_, ext, err := detectCompression(path)
-
-	ctype := "application/x-gtar"
-	if err != nil {
-		ctype = "application/octet-stream"
-	}
-
```

Comment:
On Fri, Mar 13, 2015 at 03:49:42PM -0700, St�phane Graber wrote:
> > @@ -547,20 +546,19 @@ func imageExport(d *Daemon, r *http.Request) Response {
> >  	path := shared.VarPath("images", hash)
> >  	filename := imgInfo.Filename
> >  
> > -	// test compression, for content type header
> > -	// if unknown compression we send standard header
> > -	_, ext, err := detectCompression(path)
> > -
> > -	ctype := "application/x-gtar"
> > -	if err != nil {
> > -		ctype = "application/octet-stream"
> > -	}
> > -
> 
> Why are we dropping setting Content-Type here?

It's autodetected by http.ServeContent, along with a few others.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/389/files#r26428815